CHANGELOG.md

```diff
@@ -63,6 +63,7 @@
 ### Jsonnet
 
 * [CHANGE] Remove support to set Redis as a cache backend from jsonnet. #9677
+* [CHANGE] Rollout-operator now stores scaling operation metadata in a Kubernetes configmap. #9699
```

Comment:
Maybe elaborate on this a bit, why people would care, if they should expect things to work differently (I think the answer to this is "no"), etc.